tests/utils/waitForAll.test.tsx

```diff
@@ -85,12 +86,77 @@ it('waits for two async atoms', async () => {
 
   jest.runOnlyPendingTimers()
 
-  await findByText('num1: 1')
-  await findByText('num2: 2')
+  await findByText('num1: 1, num2: 2')
```

Comment:
We should test with one `findByText` to avoid edge cases. (or, use `waitFor` and multiple `getByText`s.)